.github/workflows/README.md

```diff
@@ -35,27 +68,27 @@ The CD Workflow is responsible for automatically deploying new code changes to d
 
 
 
-**Workflow Trigger and Jobs**
-
-The CD Workflow is triggered by manual trigger through workflow dispatch, allowing developers to initiate the deployment process on demand.
-
-The CD Workflow consists of several jobs, each responsible for a specific deployment task. For instance, the `deploy-to-dev` job deploys the dev build to Firebase, while the `promote-artifact` job promotes the release build to the specified Google Play track.
-
-## **Dependency Updates workflow**
+**Version Code**
 
-Updates project dependencies using Dependabot, an automated dependency management tool, ensuring that the project always uses the latest stable versions of its dependencies.
+The version code is generated from 3 things:
+1. A "base" version code of `10000000`. For a time the version code was derived from unix time / 1000. This created a always increasing number, but was otherwise not so useful. Setting a base allows us to establish a clean "floor" for the version code. If the workflow runs ever exceed the base, first of all go us, and 2nd the base can be removed. 
+2. The `run number` of the workflow. This tells us which run a build came from and is auto incrementing. 
+3. The last two digits are reserved for the `run attempts`. This tells us how many times a specific workflow was run. If more than 99 attempts are made (something is wrong) the build will fail and you need to start a new workflow run. 
```

Comment:
Do we really need this? If a workflow fails, we'll either do 1-2 more attempts to make sure it was not a fluctuation or fix an issue and do another workflow run. In any case the build will always come from the last workflow attempt. Or am I missing something?

Reply:
It's a good questions @BurningAXE. I did it because it would prevent deployment collisions and make it explicit where exactly the build came from. Without it if someone runs the pipeline and it succeeds then for some reason the pipeline is re-run again (which is doable in the github UI) the re-run would fail the deployment. And if someone looked at the build name they'd come to the pipeline run but then have to search through to find which attempt it came from. 

 Those are small things though. So perhaps is just best to drop it for simplicity?